src/scripts/main.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+const items = document.querySelectorAll('.list-item');
+const image = document.querySelector('#largeImg');
+
+for (let i = 0; i < items.length; i++) {
+  items[i].addEventListener('click', (e) => {
+    e.preventDefault();
+
+    switch (i) {
+      case 0: image.src = './images/landscape-first.png';
+        break;
+      case 1: image.src = './images/landscape-second.png';
+        break;
+      case 2: image.src = './images/landscape-third.png';
+        break;
+      case 3: image.src = './images/landscape-fourth.png';
+        break;
+      case 4: image.src = './images/landscape-fifth.png';
+        break;
+    };
+  });
```

Comment:
You'll add each image in this `switch..case`? Just get `href` attribute from closest link tag

Reply:
there are images in different resolutions to small and big pictures, so if I take src from the closest it will look like this
![image](https://user-images.githubusercontent.com/98328152/174644683-51a9bcff-5b8f-4f91-9cfb-53114f96e6d3.png)

Reply:
You have nothing to do with resolution in this task, all you need - is reassign img src on bigImg. Remove switch..case